spec/system/spending_kudos_spec.rb

```diff
@@ -16,6 +16,7 @@
     fill_in 'Title', with: kudo.title
     fill_in 'Content', with: kudo.content
     select employee_nr_2.email, from: 'kudo_receiver_id'
+    select employee.giver_kudos.last.company_value.title, from: 'kudo_company_value_id'
```

Comment:
this looks a bit confusing, consider something like 
```suggestion
    select CompanyValue.last.title, from: 'kudo_company_value_id'
```